src/locales/en.xlf

```diff
@@ -8,47 +8,125 @@
           <context context-type="sourcefile">src/app/app.component.html</context>
-          <context context-type="linenumber">32</context>
+          <context context-type="linenumber">29</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/imprint/imprint.component.html</context>
+          <context context-type="linenumber">2</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="5744023940242558359" datatype="html">
+        <source>Dresden Internet Exchange</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/home/home.component.html</context>
+          <context context-type="linenumber">1</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="5913446743299683135" datatype="html">
+        <source>Keep Local Traffic Local.</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/home/home.component.html</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="3670532182581004841" datatype="html">
+        <source>Socials</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/home/home.component.html</context>
+          <context context-type="linenumber">41</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="5156047489151158244" datatype="html">
+        <source>Work in progress</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/imprint/imprint.component.html</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/looking-glass/looking-glass.component.html</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/privacy-policy/privacy-policy.component.html</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="2649252321173430744" datatype="html">
+        <source>Not Found</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/pages/not-found/not-found.component.html</context>
+          <context context-type="linenumber">2</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="7353139189504330359" datatype="html">
+        <source>How about jumping back to the <x id="START_LINK" ctype="x-a" equiv-text="&lt;a routerLink=&quot;/&quot;&gt;"/>home<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/> page?</source>
```

Comment:
How about jumping back ~~to the~~ home ~~page~~?

Keine Ahnung, aber ich würde sagen, dass das besser klingen würde. Da das ja die source Datei ist, musst du das direkt im Code anpassen.

Reply:
Jo das müssen wir nochmal evaulieren. Will bloß nicht das sich das so anhört wir "Lass es sein mit dem Webseitenbesuch und Dampf nach Hause ab."  (Komplett überspitzt) Aber ich glaube das ergibt sich Kontexttechnisch...

Reply:
ja keine ahnung, mach doch was du willst